src/core/src/FabricMaterial.cpp

```diff
@@ -584,15 +584,15 @@ void FabricMaterial::setImageryLayerValues(
     const pxr::TfToken& textureAssetPathToken,
     const TextureInfo& textureInfo,
     uint64_t texcoordIndex,
-    float alpha) {
+    double alpha) {
     setTextureValuesCommon(imageryLayerPath, textureAssetPathToken, textureInfo, texcoordIndex);
     setImageryLayerAlphaValue(imageryLayerPath, alpha);
 }
 
-void FabricMaterial::setImageryLayerAlphaValue(const omni::fabric::Path& imageryLayerPath, float alpha) {
+void FabricMaterial::setImageryLayerAlphaValue(const omni::fabric::Path& imageryLayerPath, double alpha) {
     auto srw = UsdUtil::getFabricStageReaderWriter();
     auto alphaFabric = srw.getAttributeWr<float>(imageryLayerPath, FabricTokens::inputs_alpha);
-    *alphaFabric = alpha;
+    *alphaFabric = static_cast<float>(alpha);
```

Comment:
Again, I'm curious about what the reason to use a double for alpha if it gets cast back to a float.